garrysmod/lua/includes/modules/worldtip.lua

```diff
@@ -0,0 +1,56 @@
+
+local IsValid = IsValid
+local SysTime = SysTime
+local isvector = isvector
+
+
+/*
+	Name: worldtip
+	Desc: Handles world/entity-based tooltips
+*/
+module( "worldtip" )
+
+local WorldTips = {}
+
+function GetTable()
+	return WorldTips
+end
+
+function Add( name, tgt, text, len, col )
+
+	local ent, pos = IsValid( tgt ) && tgt
```

Comment:
(You selected the line correctly)
*ent* either becomes *tgt* or *false* depending on the validity of *tgt*. *pos* is localised as *nil* to be initialised later. It's effectively the same as:
```lua
local ent = IsValid(tgt) && tgt || false
local pos
```
Just to note, I've written a system for spacing tooltips out and for per-frame calls so that AddWorldTip can be deprecated; I'll push it later this week once I've checked it over for issues.